internal/colorize/cropped_test.go

```diff
@@ -13,52 +13,52 @@ func Test_GetCroppedText(t *testing.T) {
 	tests := []struct {
 		name string
 		args args
-		want []Entry
+		want []CroppedLine
```

Comment:
I think this is why the unit tests are failing
```suggestion
		want CroppedLines
```